proto/metapb.proto

```diff
@@ -127,6 +127,7 @@ message Region {
     bool is_in_flashback  = 7;
     // The start_ts that the current flashback progress is using.
     uint64 flashback_start_ts = 8;
+    Peer leader = 9;
```

Comment:
It's better to add a new protobuf message in pdpb.proto that wraps `metapb.Region` and `metapb.Peer` for better extensibility.
`metapb.Region` must only include states that need to be persisted in TiKV side, so "leader" should not be added here.

Reply:
It will cause breaking changes on the PD side. BTW, I think the purpose of metapb.Region are not the same on PD and TiKV.

Reply:
+1, store `leader` in region meta is a little weird from the perspective of TiKV. Can PD store a separate key value for leader info?

Reply:
`leader` won't assign any value on the TiKV side.

Reply:
I know, if there is no other better solution. I can accept reluctantly.